modules/rte_mis_core/src/Plugin/Block/SiteLogoSectionBlock.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+namespace Drupal\rte_mis_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a 'Site log Section' block with image and text fields.
+ *
+ * @Block(
+ *   id = "site_logo_section_block",
+ *   admin_label = @Translation("Site Logo Section"),
+ * )
+ */
+class SiteLogoSectionBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entityTypeManager) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entityTypeManager;
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $config = $this->getConfiguration();
+    $build = [];
+
+    $heading = isset($config['site_logo_heading']) && $config['site_logo_heading'] !== '' ? $config['site_logo_heading'] : 'Government Of India';
+    $subtext = isset($config['site_logo_subtext']) && $config['site_logo_subtext'] !== '' ? $config['site_logo_subtext'] : 'School Education Department Right to Education(RTE) Portal';
+    $logoImg = $config['site_logo_image'] ?? '';
+
+    $build['heading']['#markup'] = '<h2>' . $heading . '</h2>';
+    $build['sub_heading']['#markup'] = '<p>' . $subtext . '</p>';
+
+    if (empty($logoImg[0])) {
+      // Load and display the default image from the assets folder.
+      $defaultImage = '/profiles/contrib/rte_mis/modules/rte_mis_core/asset/img/Emblem_of_India.svg';
```

Comment:
Please don't use the static path, you can use the Drupal service to generate the path.

Reply:
ok